torch/package/package_exporter.py

```diff
@@ -202,8 +206,11 @@ def __init__(
         else:  # is a byte buffer
             self.buffer = f
 
-        self.zip_file = torch._C.PyTorchFileWriter(f)
-        self.zip_file.set_min_version(6)
+        self.zip_file = zip_file_reader_type(f)
+        assert isinstance(self.zip_file, TorchScriptPackageZipFileWriter)
```

Comment:
asserting here seems to make zip_file_reader_type useless since you can only have a single type, is this intentional?

Reply:
This is one of those make mypy happy type of assertions :( Is there a better way to do this?